students/rimlee_talukdar/lesson01/assignment/test_unit.py

```diff
@@ -0,0 +1,100 @@
+import sys
```

Comment:
Excellent work! 

Ideally, you should try to not have the "sys.path.insert(0, './inventory_management')"

You can fix this by making your imports consistent so that you assume "assignment" is the root folder. Also when you run pytest, make sure you are running "python -m pytest ... "